index.html

```diff
@@ -0,0 +1,747 @@
+    <div class="csfsgs">Go to Home Page >></div>
+    </div>
+    <div class="footer-section">
+        <ul style="margin-left: 15rem;">
```

Comment:
Please, remove inline styling, it is considered a bad practice, move it to your CSS files.